modules/otel-collector-components.adoc

```diff
@@ -651,6 +651,46 @@ The Debug exporter prints traces and metrics to the standard output.
 ----
 <1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
 
+[id="load-balancing-exporter_{context}"]
+=== Load Balancing Exporter
+
+The Load Balancing Exporter consistently exports spans, metrics, and logs according to the `routing_key` configuration.
+
+:FeatureName: The Load Balancing Exporter
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.OpenTelemetry Collector custom resource with an enabled Load Balancing Exporter
+[source,yaml]
+----
+# ...
+  config: |
+    exporters:
+      loadbalancing:
+        routing_key: "service" # <1>
```

Comment:
For routing_key This functionality is currently enabled only for trace pipeline types. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/loadbalancingexporter#configuration